### ISSUE TITLE
JBoss Logging instrumentation doesn't capture string substitution values in log messages

### ISSUE THREAD

**Issue by jasonjkeller** [newrelic/newrelic-java-agent#1167]
When using string substitution with JBoss Logging APIs (as shown below) I would expect to see the log message captured with the substituted values in place (i.e. `This is a INFO JBoss log string1 : string2`). Instead, what is being reported is `This is a INFO JBoss log %s : %s`

```
        String s1 = "string1";
        String s2 = "string2";

        LOGGER.infof("This is a INFO JBoss log %s : %s", s1, s2);
```

It looks like this is a gap in the [jboss.logging](https://github.com/newrelic/newrelic-java-agent/tree/main/instrumentation/jboss-logging) instrumentation that should be addressed. 

<img width="870" alt="jboss-log-gap" src="https://user-images.githubusercontent.com/3496648/221960211-b2c8fb8d-7d5e-4bd6-b885-4d10b98f5bf8.png">

**Comment by danbaie** [comment]
Is there any news on this?

**Comment by jasonjkeller** [comment]
@danbaie The fix for this has been merged and will be included in the 8.8.0 Java agent release targeted for mid-December.
